packages/lib/src/utils/Transactions.ts

```diff
@@ -35,14 +42,14 @@ const TRUFFLE_DEFAULT_GAS_PRICE = BN(100000000000);
  * @param txParams other transaction parameters (from, gasPrice, etc)
  * @param retries number of transaction retries
  */
-export async function sendTransaction(contractFn, args = [], txParams = {}, retries = RETRY_COUNT) {
-  await fixGasPrice(txParams)
+export async function sendTransaction(contractFn: GenericFunction, args: any[] = [], txParams: any = {}, retries: number = RETRY_COUNT) {
```

Comment:
I'm pretty sure `args` will always be an array of strings. Could you please check it out?
Also, the function's return type is missing.

Reply:
Fixed `args: string[]`, but the return function is unknown. Setting it as `any` explicitly tho.